packages/material-ui-lab/src/SpeedDialAction/SpeedDialAction.js

```diff
@@ -116,6 +117,10 @@ SpeedDialAction.propTypes = {
    * @ignore
    */
   open: PropTypes.bool,
+  /**
+   * Make the tooltip always visible.
+   */
+  tooltipAlwaysOpen: PropTypes.bool,
```

Comment:
Perhaps this should be an `@ignore`d prop controlled by the SpeedDial? Thoughts?

Either way, `tooltipOpen ` should be sufficient for the prop name.